docs/source/guide/tasks.md

```diff
@@ -191,10 +192,21 @@ this is a second task
 If you want to import entire plain text files without each line becoming a new labeling task, customize the labeling configuration to specify `valueType="url"` in the Text tag. See the [Text tag documentation](tags/text.html)
 
 ## Import data from a local directory
+
 To import data from a local directory, you have two options:
-- Add the file directory as a source or target [local storage](storage.html#Local-storage) connection in the Label Studio UI. 
 - Run a web server to generate URLs for the files, then upload a file that references the URLs to Label Studio. 
+- Add the file directory as a source or target [local storage](storage.html#Local-storage) connection in the Label Studio UI.
+
+Reference tasks in a local directory using the following syntax in the Label Studio JSON format. For example, to reference an image file `1.png`: 
+```
```

Comment:
why do we need this duplicated with https://github.com/heartexlabs/label-studio/pull/797/files#diff-446481c51c42df4652b3b424cf7c3f1e69f4cfea206d82a3ec68e51e7f7474cdR296 ?

Reply:
good point, I'll remove it from this spot. I think I had it here so that people using the local web server method would have this reference, but I think that will be less common anyway.